modules/xpfunmac.cmake

```diff
@@ -872,9 +864,10 @@ function(xpGitIgnoredDirs var dir)
     OUTPUT_VARIABLE ignoredDirs
     OUTPUT_STRIP_TRAILING_WHITESPACE
     )
-  separate_arguments(ignoredDirs UNIX_COMMAND "${ignoredDirs}")
+  string(REPLACE ";" "\\\\;" ignoredDirs "${ignoredDirs}")
+  string(REPLACE "\n" ";" ignoredDirs "${ignoredDirs}")
   list(APPEND ignoredDirs ${ARGN})
-  xpListPrependToAll(ignoredDirs ${dir} ${ignoredDirs})
+  list(TRANSFORM ignoredDirs PREPEND ${dir}/)
```

Comment:
`list(TRANSFORM` is a good find! I remembered using it sometime recently so I went and found where... in internpro for palam https://isrhub.usurf.usu.edu/smanders/internpro/commit/d07f61cc2bd424ce0b2b15e86284c94ef35697f5#diff-f22573e5bfe084118cf3aceabbbafedd -- there are very likely other places in all of our cmake that could use enhancements like this, so I really appreciate you discovering a better way for doing what `xpListPrependToAll()` was doing!